example/main.tf

```diff
@@ -1,17 +1,40 @@
-################################################################################
-## defaults
-################################################################################
 terraform {
-  required_version = "~> 1.3, < 2.0.0"
+  required_version = "~> 1.5"
 
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 4.0"
+      version = "~> 5.0"
     }
   }
 }
 
 provider "aws" {
   region = var.region
 }
+
+
+
+module "tags" {
+  source  = "sourcefuse/arc-tags/aws"
+  version = "1.2.3"
+
+  environment = var.environment
+  project     = var.namespace
+
+  extra_tags = {
+    MonoRepo     = "True"
+    MonoRepoPath = "terraform/kms"
+  }
+}
+
+
+module "kms" {
+  source                  = "../" # Use the local path to your kms-module directory
+  enabled                 = var.enabled
+  deletion_window_in_days = var.deletion_window_in_days
+  enable_key_rotation     = var.enable_key_rotation
+  alias                   = var.alias
+  tags                    = module.tags.tags
+  policy                  = var.policy
```

Comment:
Where are you defining policy ?

Reply:
I haven't defined the policy in example folder, just specified the empty variable for the policy, i will add a policy in example folder

Reply:
Its added in `examplevariables.tf` now